frontend/src/routes/console.tsx

```diff
@@ -66,7 +75,15 @@ export const ConsoleRoutes = () => {
         exact
         path={ROUTES.Console.NEW_LOGOS_SIDEBAR}
         component={LogosSideBar}
-      /> */}
+      />
+      <PrivateRoute exact path={ROUTES.Console.NEW_THEME_FOOTER}>
+        <Redirect to={ROUTES.Console.NEW_THEME_FOOTER_SIDEBAR} />
+      </PrivateRoute>
```

Comment:
@giantas You don't need this route. You only need one page, that is `/newconsole/theming/footer/`.